AzureDataLake/Scripts/ProcessCCB.usql

```diff
@@ -0,0 +1,70 @@
+﻿USE DATABASE [DecisionServiceDBTest];
+
+REFERENCE ASSEMBLY DecisionServiceDBTest.[DecisionServiceExtractor];
+
+DECLARE @_appId string = "xbetdealshomespotlight";    //your appId
+DECLARE @_account string = "decisionengine";    // your account name
+
+DECLARE @_year string = "2019";  //your date: year
+DECLARE @_month string = "10";     //your date: month
+DECLARE @_day string = "14";       //your date: day
+
+DECLARE @OutputFolder string = "/" + @_appId + "_daily/";      //your output folder
+DECLARE @OutputSlimLogs bool = false;     //true if you want to have slime version of logs (all metadata without features)
+     
+DECLARE @InputFile string = "wasb://"+ @_appId +"@" + @_account + ".blob.core.windows.net/{model}/data/{year}/{month}/{day}_{filename}.json";
+
+DECLARE @OutputFileInteractions string = @OutputFolder + "interactions-" + @_year + "-" + @_month + "-" + @_day + ".csv";
+DECLARE @OutputFileDangling string = @OutputFolder + "dangling-" + @_year + "-" + @_month + "-" + @_day + ".csv";
+DECLARE @OutputFileStatistics string = @OutputFolder + "statistics-" + @_year + "-" + @_month + "-" + @_day + ".csv";
+DECLARE @OutputFileInvalid string = @OutputFolder + "invalid-" + @_year + "-" + @_month + "-" + @_day + ".csv";
+
+@Events =  
+    EXTRACT SessionId string,
+            SlotIdx int,
+            EventId string,
+            model string,
+            ParseError string,
+            day string, //virtual column
+            month string, //virtual column
+            year string, //virtual column
```

Comment:
we should use the same logic with datetime used in PR #162

Reply:
no, just because it was 2 separate PR which were done independently. will update.